CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ After this we will switch probably to real [Semantic Versioning 2.0.0](http://se
 * Fix #1695: IllegalArgumentException when Spring Boot application.yaml contains integer keys
 * Fix #797: spring-boot generator can not handle multi-profile configuration
 * Fix #1751: Build Names are suffixed with -*s2i regardless of build strategy
+* Minor improvement: Check maven.compiler.target property for base image detection.
```

Comment:
Not related to this PR but everything seems to be italic here due to `*` added in the line above. Could you please trim it?